app/src/main/java/com/vitorpamplona/amethyst/ui/note/ReactionsRow.kt

```diff
@@ -1438,13 +1438,14 @@ fun showCount(count: Int?): String {
     return when {
         count >= 1000000000 -> "${(count / 1000000000f).roundToInt()}G"
         count >= 1000000 -> "${(count / 1000000f).roundToInt()}M"
-        count >= 1000 -> "${(count / 1000f).roundToInt()}k"
+        count >= 10000 -> "${(count / 1000f).roundToInt()}k"
```

Comment:
I think the Divisor here must be updated as well.

Reply:
Can you elaborate?
Changing the divisor from a standard metric unit (1000) to match the size at which its being used (10000) would be erroneous and I'm not understanding what you mean otherwise.